garage/contrib/ros/docker/sawyer/docker-entrypoint.sh

```diff
@@ -0,0 +1,253 @@
+#!/bin/bash
+
+ORANGE='\033[0;33m'
```

Comment:
Please do not install anything in an entrypoint. That's what Dockerfiles are for. You're not even guaranteed internet access when running the Dockerfile.